Sources/ApexyAlamofire/AlamofireClient.swift

```diff
@@ -107,8 +107,12 @@ open class AlamofireClient: Client {
                 completionHandler: { (response: DataResponse<Data, AFError>) in
 
                     let result = APIResult<T.Content>(catching: { () throws -> T.Content in
-                        let data = try response.result.get()
-                        return try endpoint.content(from: response.response, with: data)
+                        do {
+                            let data = try response.result.get()
+                            return try endpoint.content(from: response.response, with: data)
+                        } catch {
+                            throw error.unwrapAlamofireValidationError()
```

Comment:
will `unwrapAlamofireValidationError` be reused in other places ?

or is it possible to write a casting error only in this place ?

```swift
} catch let error as AFError {                            
    throw error.underlyingError
}
```

Reply:
It will be useful in `RequestInterceptor`. The `retry` method will receive non unwrapped error `responseValidationFailed ` and for some checks may need to get underlyingError.